invenio_users_resources/services/permissions.py

```diff
@@ -48,6 +48,7 @@ class UsersPermissionPolicy(BasePermissionPolicy):
         SystemProcess(),
     ]
     can_read_details = [UserManager, Self(), SystemProcess()]
+    can_read_all = [UserManager, SystemProcess()]
```

Comment:
`UserManager` role covers `administration-moderation` role. Should we also add support for `administration-access` role? It should be `Administration()`, but I can be wrong.

Reply:
Isn't `administration-access` role for just having rights for accessing the administration panel? The `administration-moderation` role is for user management.